SwiftVocabulary/SwiftVocabulary/WordsTableVC.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  WordsTableVC.swift
+//  SwiftVocabulary
+//
+//  Created by Norlan Tibanear on 5/20/20.
+//  Copyright © 2020 Norlan Tibanear. All rights reserved.
+//
+
+import UIKit
+
+class WordsTableVC: UITableViewController, UITextFieldDelegate {
```

Comment:
Just so you know, some people/places prefer to spell out ViewController instead of VC. I prefer spelling it out in class names, but then using the shorthand for instances. This seems to be the general preference of the instructors here at Lambda as well, as they always have ViewController spelled out in the class names.